docs/source/api_reference/featurizers.rst

```diff
@@ -31,6 +31,10 @@ Molecule Featurizers
 
 These featurizers work with datasets of molecules.
 
+All featurizers are based on an abstract parent class (:code:`dc.feat.Featurizer`). This is used to create abstract base classes for a few general types of featurization, including (:code:`MolecularFeaturizer`), (:code:`MaterialCompositionFeaturizer`), etc. 
```

Comment:
I think these will be more apt in the `Base Featurizers (for develop)` section of docs, just above `MolecularFeaturizer` section (https://github.com/deepchem/deepchem/blob/master/docs/source/api_reference/featurizers.rst#molecularfeaturizer)